### ISSUE TITLE
Xwayland windows cause weird graphical glitch.

### ISSUE THREAD

**Issue by snakedye** [swaywm/sway#5805]
### Please read the following before submitting:
- Please do NOT submit bug reports for questions. Ask questions on IRC at #sway on irc.freenode.net.
- Proprietary graphics drivers, including nvidia, are not supported. Please use the open source equivalents, such as nouveau, if you would like to use Sway.
- Problems with the Wayland version of Firefox are likely to be Firefox bugs. Start by submitting your issue to the Firefox Bugzilla and come back here only after they confirm otherwise.

### Please fill out the following:
- **Sway Version:**
  - sway 1.5

- **Debug Log:**
  - Run `sway -d 2> ~/sway.log` from a TTY and upload it to a pastebin, such as gist.github.com.
https://gofile.io/d/9oWhQO

- **Configuration File:**
https://gofile.io/d/wEBzkh

- **Stack Trace:**
  - This is only needed if sway crashes.
  - If you use systemd, you should be able to open the coredump of the most recent crash with gdb with
    `coredumpctl gdb sway` and then `bt full` to obtain the stack trace.
  - If the lines mentioning sway or wlroots have `??` for the location, your binaries were built without debug symbols. Please compile both sway and wlroots from source and try to reproduce.

- **Description:**
  - The steps you took in plain English to reproduce the problem.
https://gofile.io/d/mmoijh
Open Chromium (or an electron app like Caprine), open another window in this case Alacritty (it will be tiled next to chromium) and kill that window.

**Comment by Xyene** [comment]
I'm afraid these sorts of graphical glitches are outside of Sway's domain; it just renders the buffers Xwayland provides it. To some extent, graphical glitches in Xwayland apps _are_ expected. You could try playing with different (newer) Xwayland versions.
